eurofaang_ri_be/tna/views.py

```diff
@@ -22,14 +21,16 @@ class TnaProjectViewSet(viewsets.ModelViewSet):
     ordering_fields = ['project_title', 'principal_investigator__first_name', 'principal_investigator__last_name',
                         'associated_application', 'id']
 
+    def get_queryset(self):
+        if self.request.user.role == 'PI':
+            return TnaProject.objects.filter(tna_owner=self.request.user)
+        else:
+            return TnaProject.objects.filter(additional_participants=self.request.user)
+
```

Comment:
User can be "PI" and "additional participant" at the same time, so this filter won't work